MediaBrowser.Providers/MediaInfo/AudioFileProber.cs

```diff
@@ -104,8 +107,50 @@ public partial class AudioFileProber
             }
 
             var libraryOptions = _libraryManager.GetLibraryOptions(item);
+            bool foundLUFSValue = false;
 
-            if (libraryOptions.EnableLUFSScan)
+            if (libraryOptions.UseReplayGainTags)
+            {
+                using (var process = new Process()
+                {
+                    StartInfo = new ProcessStartInfo
+                    {
+                        FileName = _mediaEncoder.ProbePath,
+                        Arguments = $"-hide_banner -i \"{path}\"",
+                        RedirectStandardOutput = false,
+                        RedirectStandardError = true
+                    },
+                })
+                {
+                    try
+                    {
+                        process.Start();
+                    }
+                    catch (Exception ex)
+                    {
+                        _logger.LogError(ex, "Error starting ffmpeg");
+
+                        throw;
+                    }
+
+                    using var reader = process.StandardError;
+                    var output = await reader.ReadToEndAsync(cancellationToken).ConfigureAwait(false);
+                    cancellationToken.ThrowIfCancellationRequested();
+                    MatchCollection split = ReplayGainTagRegex().Matches(output);
```

Comment:
Why use Matches if you only expect one? The code would be a little clearer with just `.Match(output)`